ServiceAPIExtensions/Controllers/ContentAPiController.cs

```diff
@@ -103,6 +103,7 @@ public static ExpandoObject ConstructExpandoObject(IContent c, string Select=nul
         }
 
         public static ExpandoObject ConstructExpandoObject(IContent c, bool IncludeBinary,string Select=null, Encryption encryption = Encryption.SHA256)
+            //Freek: we don't seem to use encryption from an external endpoint anywhere. Is it necessary? Is it tested?

```

Comment:
This has been implemented as a precaution, in case we do want to specify this in the (near) future.